src/api/public/apidocs-new/paths/search_owner.yaml

```
@@ -0,0 +1,100 @@
+get:
+  summary: Search for the default responsible person or group (the owner).
+  description: |
+    Search for the default responsible person or group (the owner) by a user, group
+    or binary name.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: query
+      name: binary
+      schema:
+        type: string
+      description: |
+        The name of a build binary. Either the `binary`, `user` or `group`
+        parameter must be provided.
+    - in: query
+      name: user
+      schema:
+        type: string
+      description: |
+        The login of a user. Either the `binary`, `user` or `group`
+        parameter must be provided.
+    - in: query
+      name: group
+      schema:
+        type: string
+      description: |
+        The title of a group. Either the `binary`, `user` or `group`
+        parameter must be provided.
+    - in: query
+      name: limit
+      schema:
+        type: string
+      description: |
+        Limit the number of returned results. This is only available for
+        the search with the `binary` parameter. The default is set to `1`,
+        use `0` to receive all results or `-1` for the deepest match.
+    - in: query
+      name: devel
+      schema:
+        type: boolean
```

```suggestion
        type: string
```
Citing [the documentation](https://swagger.io/docs/specification/data-models/data-types/#boolean): _Note that truthy and falsy values such as "true", "", 0 or null are not considered boolean values._

---

done